packages/pxweb2-ui/src/lib/components/BottomSheet/BottomSheet.module.scss

```diff
@@ -0,0 +1,112 @@
+@use '../../../../style-dictionary/dist/scss/fixed-variables.scss' as fixed;
+
+.bottomSheet {
+  display: flex;
+  width: 100%;
+  height: 95vh;
```

Comment:
This might need to be tested on mobile browsers. As the viewport changes when scrolling on certain browsers. It used to be like that at least, maybe it's changed now. 

Many mobile browsers used to change the viewport height when the user scrolls. Because the interface of the browser gets shown when scrolling, and gets hidden incertain cases. That used to change the viewport height. That makes the site move the elements around when the user stops scrolling.

Reply:
Fixed the other comments. We have tested on Iphone and samsung. The height seems to be stable on both.